README.md

```diff
@@ -39,8 +39,8 @@ RegionState:SetVisibleWhenOutsideRegions("Region3")
 
 --Make it so region 1 can see 2 and 3.
 --ConnectRegions makes it so both regions can see each other.
-RegionStateObject:ConnectRegions("Region1", "Region2")
-RegionStateObject:ConnectRegions("Region1", "Region3")
+RegionState:ConnectRegions("Region1", "Region2")
+RegionState:ConnectRegions("Region1", "Region3")
```

Comment:
(No change) This was a good find. I completely missed that when copying from the unit test.